client/src/components/FrequentlyAskedQuestions.tsx

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { Icon, Link, Stack, StackItem, Text } from '@fluentui/react';
+import {
+  linkStyles,
+  font16pxStyle,
+  btnStackStyles,
+  newWindowIconWrapper,
+  textDecorationNone
+} from '../styles/Home.styles';
+
+export const FrequentlyAskedQuestions: React.VoidFunctionComponent = () => {
+  return (
+    <>
+      <Text styles={font16pxStyle}>Frequently asked questions</Text>
+      <Stack styles={btnStackStyles}>
+        <Link
+          tabIndex={0}
+          data-testid="bookingsSetupLink"
```

Comment:
this is not being used anywhere, we can remove it

Reply:
done